src/static/js/modules/external-site.js

```diff
@@ -0,0 +1,78 @@
+/* ==========================================================================
+   External Site Initialization
+   Used on at least `/external-site/`.
+   ========================================================================== */
+
+'use strict';
+
+var $ = require( 'jquery' );
+var handlebars = require( 'handlebars' );
+
+// Instantiate ExternalSite class
+// if external-site_container present on page.
+function init() {
+  var $externalSiteContainer = $( '.external-site_container' );
+  if ( $externalSiteContainer.length ) {
+    new ExternalSite( $externalSiteContainer );
+  }
+}
+
+/**
+* ExternalSite constructor.
+* @class
+* @param {HTMLElement} $element DOM Element.
+*/
+function ExternalSite( $element ) {
+  this.$element = $element;
+  this.init();
+}
+
+// ExternalSite methods and shared properties.
+ExternalSite.prototype = {
+
+  duration: 5,
+
+  interval: 1000,
+
+  template: handlebars
+            .compile( '<span class=\'external-site_reload-duration\'>' +
+                        '{{ duration }}</span>' +
+                      ' second{{ plurality }}'
+                    ),
+
+  init: function() {
+    var self = this;
+    this.$durationEl = this.$element
```

Comment:
You've got `this` defined as `self`. Is it worth using throughout this function?